config/recipes/cert-manager/webhook.asciidoc

```diff
@@ -0,0 +1,168 @@
+[id="{p}-webhook"]
```

Comment:
Should we not add this file in our current documentation ? Not sure I would look into the `recipes` directory if I was looking for information about how to configure the webhook with the cert manager.

Reply:
I'm okay with moving it there. I think I had it here with the idea that this would be a rarely used option once we have automatic configuration enabled